### PR TITLE
Hide events section

### PR DESCRIPTION
**Pull request by sniff**

### CODE REVIEW

File: index.adoc [hunk 1]
```diff
@@ -183,7 +183,7 @@ include::chapters/common-data-types.adoc[]
 include::chapters/common-headers.adoc[]
 include::chapters/proprietary-headers.adoc[]
 include::chapters/api-operation.adoc[]
-include::chapters/events.adoc[]
+// include::chapters/events.adoc[]
```
Comment (florian-stefan) [inline index.adoc]: Why not deleting all files?

Comment (sniff) [reply index.adoc]: We have a fork of Zalando repo and deleting them will make rebases very hard. If we decide not to rebase forever, then I'm fully up to delete them.

File: index.adoc [hunk 2]
```diff
@@ -178,7 +178,6 @@ include::chapters/common-data-types.adoc[]
 include::chapters/common-headers.adoc[]
 include::chapters/proprietary-headers.adoc[]
 include::chapters/api-operation.adoc[]
-include::chapters/events.adoc[]
```
Comment (apankowski) [inline index.adoc]: nitpicking: have you maybe checked if there are any references to the guidelines in the events section in other sections?

**Comment by apankowski** [comment]
👏 one of the cleanest PRs I've seen lately :)

File: index.adoc (hunk 2 above)
Comment (sniff) [reply index.adoc]: done!

**Comment by sniff** [comment]
There are some references to events in the Changelog, but I think that we should update it in the separate pull request.

**Pull request merged**